moveit_ros/moveit_servo/src/servo.cpp

```diff
@@ -194,51 +191,52 @@ bool Servo::validateParams(const servo::Params& servo_params) const
     RCLCPP_ERROR(LOGGER, "When publishing a std_msgs/Float64MultiArray, "
                          "you must select positions OR velocities."
                          "Check the parameters YAML file used to launch this node.");
-    params_valid = false;
+    return false;
   }
 
   if (servo_params.scene_collision_proximity_threshold < servo_params.self_collision_proximity_threshold)
   {
     RCLCPP_ERROR(LOGGER, "Parameter 'self_collision_proximity_threshold' should probably be less "
                          "than or equal to 'scene_collision_proximity_threshold'."
                          "Check the parameters YAML file used to launch this node.");
-    params_valid = false;
+    return false;
+  }
+
+  if (!servo_params.active_subgroup.empty() && servo_params.active_subgroup != servo_params.move_group_name &&
+      !joint_model_group->isSubgroup(servo_params.active_subgroup))
+  {
+    RCLCPP_ERROR(LOGGER,
+                 "The value '%s' Parameter 'active_subgroup' does not name a valid subgroup of joint group '%s'.",
+                 servo_params.active_subgroup.c_str(), servo_params.move_group_name.c_str());
+    return false;
   }
 
-  return params_valid;
+  return true;
 }
 
 bool Servo::updateParams()
```

Comment:
Need to make sure Servo is paused before doing this for a couple reasons:

- Don't want to have these operations in the control loop
- I think it's good from a safety perspective to stop the other joints beside this subgroup. Pausing should do that